src/logic/calculate.js

```diff
@@ -0,0 +1,38 @@
+import operate from './operate';
+
+export default function calculate(data, ButtonName) {
```

Comment:
[Required] Please follow the JavaScript naming convention. **ButtonName** should be camelCase.